core/web/webauthn_controller.go

```diff
@@ -42,7 +43,7 @@ func (c *WebAuthnController) BeginRegistration(ctx *gin.Context) {
 
 	webAuthnConfig := c.App.GetWebAuthnConfiguration()
 
-	options, err := c.inProgressRegistrationsStore.BeginWebAuthnRegistration(user, uwas, webAuthnConfig)
+	options, err := c.inProgressRegistrationsStore.BeginWebAuthnRegistration(*user, uwas, webAuthnConfig)
```

Comment:
The change here was because we get the user from the session context instead, which looks to have been set up as a pointer which propagates around starting from `core/web/auth/auth.go` , `core/services/webhook/authorizer.go` has some logic, worth the refactor here?

Reply:
The potential nil case is handled above (L32), which would occur if a bug was introduced in the gin middleware where user is not set properly, which would raise errors in the tests